### ISSUE TITLE
Geocache details page implementation

### ISSUE THREAD

**Issue by visitharri** [PromZona-Student/Hive-Geocaching#17]

**Comment by visitharri** [comment]
Decided in programming team to be postponed after #15

**Comment by visitharri** [comment]
There was a mistake done by github-project-automation bot, when it transfered this to Done. It has been only started, not even committed. I don't know, how I got this #17 linked by #7, but it is absolutely a mistake.
